dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -155,10 +155,14 @@ function accept ()
    osd_msg[3] = osd_msg[3]:format( paying_faction:name() )
    misn.osdCreate( osd_title, osd_msg )
 
+
    last_sys = system.cur()
    job_done = false
    target_killed = false
 
+   misn.createLog(misn_title:format( misn_level[level], missys:name() ), "Pirate hunt",0)
+   misn.appendLog("Pirate mission in " .. missys:name() .. " accepted on " .. last_sys:name() )
+
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.